model/src/main/java/uk/gov/hmcts/payment/api/service/PaymentGroupServiceImpl.java

```diff
@@ -152,13 +152,13 @@ private static Predicate getPredicate(
         String ccdCaseNumber, String tablejoin) {
         List<Predicate> predicates = new ArrayList<>();
 
-        if (ccdCaseNumber != null & tablejoin.equals("payments")) {
+        if (ccdCaseNumber != null && tablejoin.equals("payments")) {
```

Comment:
Might be a good idea to use constants for the Table names if SONAR isn't complaining

Reply:
Done